components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -3871,6 +3897,15 @@ public void updatePermissionListOfGroup(String groupName, String[] permissionToA
         return filteredFlatAttributeMap;
     }
 
+    private Map<String, Attribute> getFilteredEnterpriseUserSchemaAttributes(Map<ExternalClaim, LocalClaim>
+                                                                                     scimClaimToLocalClaimMap) {
+
+        return scimClaimToLocalClaimMap.entrySet().stream()
```

Comment:
can scimClaimToLocalClaimMap be null? if so add a null check

Reply:
It can't be null. Empty map is returned from previous method.